File: Codinsa2015.Client/Codinsa2015.Client/Views/Clank/ClientJava/src/net/codinsa2015/SignalType.java
package net.codinsa2015;
public enum SignalType
{
	ComingToPosition(0),
	AttackEntity(1),
	DefendEntity(2);
	int _value;
	SignalType(int value) { _value = value; } 
	public int getValue() { return _value; }
	public static SignalType fromValue(int value) { 
		switch(value) { 
			case 0: return ComingToPosition;
			case 1: return AttackEntity;
			case 2: return DefendEntity;
		}
		return ComingToPosition;
	}
}

File: Codinsa2015/Codinsa2015/Server/Signals/SignalType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Codinsa2015.Server.Signals
{
    [Clank.ViewCreator.Enum("Enumère la liste des différents signaux existants.")]
    public enum SignalType
    {
        ComingToPosition,
        AttackEntity,
        DefendEntity,
    }
}
